src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -163,7 +169,7 @@ private void add(TaskListener listener, SCMSourceObserver observer, GHRepository
     }
 
     public interface GitHubSCMSourceAddition extends ExtensionPoint {
-        List<? extends SCMSource> sourcesFor(String checkoutCredentialsId, String scanCredentialsId, String repoOwner, String repository);
+        List<? extends SCMSource> sourcesFor(String apiUri, String checkoutCredentialsId, String scanCredentialsId, String repoOwner, String repository);
```

Comment:
This is a binary-incompatible change. Could break people upgrading from beta releases. I hope they know to upgrade all related plugins.